modules/playgroundxyzBidAdapter.js

```diff
@@ -91,24 +66,30 @@ export const spec = {
   interpretResponse: function (serverResponse, { bidderRequest }) {
     serverResponse = serverResponse.body;
     const bids = [];
+
     if (!serverResponse || serverResponse.error) {
       let errorMessage = `in response for ${bidderRequest.bidderCode} adapter`;
       if (serverResponse && serverResponse.error) { errorMessage += `: ${serverResponse.error}`; }
       utils.logError(errorMessage);
       return bids;
     }
-    if (serverResponse.tags) {
-      serverResponse.tags.forEach((serverBid, index) => {
-        const rtbBid = getRtbBid(serverBid);
-        if (rtbBid) {
-          if (rtbBid.cpm !== 0 && includes(this.supportedMediaTypes, rtbBid.ad_type)) {
-            const bid = newBid(serverBid, rtbBid, bidderRequest);
-            bid.mediaType = parseMediaType(rtbBid);
+
+    if (!utils.isArray(serverResponse.seatbid)) {
+      let errorMessage = `in response for ${bidderRequest.bidderCode} adapter`;
+      utils.logError(errorMessage += 'Malformed seatbid response');
```

Comment:
Not very important, but do you want to put a space or a `:` between `errorMessage` and `Malformed seatbid response` ?